src/OpenSage.Game/Data/W3d/W3dRgbF.cs

```diff
@@ -1,32 +1,27 @@
-﻿using System.IO;
+﻿using OpenSage.Mathematics;
+using System.IO;
 using System.Runtime.InteropServices;
 
 namespace OpenSage.Data.W3d
 {
-    [StructLayout(LayoutKind.Sequential)]
-    public struct W3dRgbF
+    public static class W3dRgbF
```

Comment:
There are already extension methods for `ReadColorRgbF` and `Write(ColorRgbF)`, so actually you can remove this whole class.